src/usb_reset/src/ros_usb_reset.cpp

```diff
@@ -0,0 +1,89 @@
+#include "usb_reset/ros_usb_reset.hpp"
+
+int num = 0;
+
+ROS_Usb_Reset::ROS_Usb_Reset(QQuickItem * parent) :
+    QQuickPaintedItem(parent),
+    topic_value(),
+    ros_ready(false),
+    devices("no devices") {
+
+}
+
+void ROS_Usb_Reset::setup(ros::NodeHandle * nh, QQmlContext *ctxt) {
+
+    //TODO: make these parameters of the component
```

Comment:
Please either: create a jira ticket for the todo, do the todo, remove the todo :)